### DISCUSSION TITLE
Module not found: Error: Can't resolve 'fs' in '/usr/src/app/node_modules/jpeg-exif/lib'

### DISCUSSION THREAD

**Discussion by krishnav3it** [diegomura/react-pdf#2552] [General]
Team!! This plugin worked fine last month but when I re-run the npm I and run the app it throws error  Module not found: Error: Can't resolve 'fs' in '/usr/src/app/node_modules/jpeg-exif/lib' . Please help me out.
Thank you

**Comment by diegomura** [comment]
Duplicated: https://github.com/diegomura/react-pdf/issues/2535

Please please please community, review existing issues before posting. I lose a lot of time deduping subjects

**Reply by krishnav3it** [reply]
Sorry , Thank u
